src/scikitlearn.jl

```diff
@@ -49,8 +49,7 @@ ScikitLearnBase.predict_log_proba(gmm::GMM, X) = log(gmmposterior(gmm, X)[1])
 ScikitLearnBase.predict_proba(gmm::GMM, X) = gmmposterior(gmm, X)[1]
 ScikitLearnBase.predict(gmm::GMM, X) =
     # This is just `argmax(axis=2)`. It's very verbose in Julia.
```

Comment:
That comment is now outdated. Could we remove it please?

Reply:
Done via a95d901728915d7347ba172182c78b1ea31a79c6